static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,192 @@
+# Data Registry
+
+We developed the `dvc get`, `dvc import`, and `dvc update` commands with the aim
+to enable reusability of any <abbr>data artifacts</abbr> (raw data, intermediate
+results, models, etc) between different projects. For example, project A may use
+a data file to begin its data [pipeline](/doc/command-reference/pipeline), but
+project B also requires this same file; Instead of
+[adding it](/doc/command-reference/add#example-single-file) it to both projects,
+B can simply import it from A.
+
+Taking this idea to a useful extreme, we could create a <abbr>project</abbr>
+that is exclusively dedicated to
+[tracking and versioning](/doc/use-cases/data-and-model-files-versioning)
+datasets (or any kind of large files) – by mainly using `dvc add` to build it.
+Such a project would not have [stages](/doc/command-reference/run), but its data
+files may be updated manually as they evolve. Other projects can then share
+these artifacts by downloading (`dvc get`) or importing (`dvc import`) them for
+use in different data processes – and these don't even have to be _DVC
+projects_, as `dvc get` works anywhere in your system.
+
+The advantages of using a data registry are:
+
+- Tracked files can be safely stored in a **centralized** remote location, with
+  the ability to create any amount of distributed copies on other remote
+  storage.
+- Several projects can **share** the same files, trusting that everyone is using
+  the same data versions.
+- Projects that import data from the registry don't need to push these large
+  files to their own [remotes](/doc/command-reference/remote), **saving space**
+  on storage – they may not even need to a remote at all, using only the local
+  <abbr>cache</abbr>.
+- Easier to manage **access control** per remote storage configured in a single
+  data registry project.
+
+A possible risk of shared data registries is that, if the source project or its
+remote storage are lost for any reason, several other projects depending on it
+may stop being reproducible. So this strategy is best when the registry is owned
+and controlled internally by the same team as the projects that employ it.
+
+## Example: A sub-optimal approach
```

Comment:
these two title create a feeling that there are two separate examples, let's think how simplify restructure them. Also, not sure that Example is a good title at all. 

It feels that we are making a tutorial out of it - all reproducible with all the steps and details. While we should be focusing on a high level scenario here. I would prepare data-registry in advance, show on a conceptual level with some links that `dvc add` is used to create a version. Then jump into `dvc import`  - again, simple conceptual example. Then, `dvc add` to update dataset in the registry. Then, `dvc update` to update it in the import. If we want to show internal details - let's use expandable sections.

Reply:
> these two title create a feeling that there are two separate examples, let's think how simplify restructure them. Also, not sure that Example is a good title at all.

Updated in 0f7b1a16.

Reply:
> ...reproducible with all the steps and details. While we should be focusing on a high level scenario here. I would prepare data-registry in advance...

Sure. Originally we talked about showing all the steps from `wget` to `dvc get` to `import`, along with how the data registries would look at each step, so thats how we ended up with this very long tutorial-like doc. Let me re-work all this then... I'll report back.

Reply:
Fixed as explained in https://github.com/iterative/dvc.org/pull/679#issuecomment-544789692